tests/test_simple.py

```diff
@@ -252,6 +252,15 @@ def test_single_image():
     test_params = psf.interp.interpolate(target)
     numpy.testing.assert_almost_equal(test_params, true_params, decimal=5)
 
+    # Test that we can make rho statistics
+    stats = piff.RhoStatistics.read(psf_file)
+    logr, rho1, rho2, rho3, rho4, rho5 = stats.stats(stars, logger=logger, max_sep=5000)
+
+    # Test that we can make the rho plots
+    rho = stats.rho[1]
+    fig, ax = stats.plot(rho)
+    import matplotlib.pyplot as plt
+    plt.show()
```

Comment:
Don't ever have a show() command in unit tests.  They stop the test and force the user to close the window.  Better would be to write it out to a file in a `plots` directory.  Then developers can open those files if they want to make sure things look pretty.  But when doing simple regression testing, we won't need to care about them.